### PR TITLE
MCKIN-30011 management command to fix discussion error 

### PR DESCRIPTION
**Pull request by mudassir-hafeez**
Management command to fix discussion error on posting a private message

### CODE REVIEW

**Comment by mudassir-hafeez** [comment]
@moeez96 please don't merge as we need go-ahead from QA team so far.

**Review by moeez96 — APPROVED** [review]

File: edx_solutions_projects/management/commands/fix_existing_groups_cohorts.py
Comment (nasirhjafri) [inline edx_solutions_projects/management/commands/fix_existing_groups_cohorts.py:11]: This could be a boolean i.e. --fix

**Review by nasirhjafri — APPROVED** [review]

**Pull request merged**
